python/py-crossbar/Portfile

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-crossbar
+version             20.8.1
+revision            0
+categories          python net
+platforms           darwin
+license             AGPL-3
+
+homepage            http://crossbar.io
+
+description         Multi protocol WAMP/WebSocket REST/HTTP MQTT application \
+                    router for microservices.
+long_description    {*}${description}
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+checksums           rmd160  c52ae7e750d1ea397c6133d6973d39360d7d7511 \
+                    sha256  c0e7a25c12fd768e8cc9be3f572d0f7399b43089b0396635f2ab609fa2855572 \
+                    size    382515
+
+python.versions     38
+
+# Patches:
+#
+#   patch-requirements-min.txt.diff:
+#       remove maximum version limit for idna and urllib3
+#
+#   patch-setup.py.diff:
+#       avoid installing LICENSE files into the Python site-packages root
+#
+patchfiles          patch-requirements-min.txt.diff \
+                    patch-setup.py.diff
```

Comment:
If these patchfiles are only needed by the python module, and not by the crossbar subport, then the `patchfiles` statement should be moved inside the `if {${subport} ni "${name} crossbar"} {` block.

Reply:
Yep, moved.